infra-as-code/bicep/ai-agent-service-dependencies.bicep

```diff
@@ -63,8 +63,19 @@ module deployAzureAISearchService 'ai-search.bicep' = {
   }
 }
 
+@description('Deploy Azure AI Search instance for the Azure AI Foundry Agent Service (dependency). This is used when a user uploads a file to the agent, and the agent needs to search for information in that file.')
+module deployUserManagedIdentity 'ai-agent-user-managed-identity.bicep' = {
+  name: 'agentUserManagedIdentityDeploy'
```

Comment:
Shouldn't the dependency modules here be passed the user-assigned managed identity so they can do role assignments? (like they are doing for the "debug" user for example)